src/test/daml/Daml/Finance/Instrument/Option/Test/Util.daml

```diff
@@ -93,7 +93,7 @@ electAndVerifyPaymentEffects readAs today amount instrument issuer custodian ele
         description = "Rule to lifecycle a generic instrument"
 
     let
-      electorIsOwner = True
+      electorIsOwner = isLong
       counterparty = issuer
       description = "election for physically settled option"
     exerciseOptionCid <- submitMulti [custodian] readAs
```

Comment:
Do we want to have the `custodian` here? Will you look at this as part of a following PR?

Reply:
Correct, I planned to fix that, possibly together with a few similar improvements, here:
https://github.com/digital-asset/daml-finance/issues/701